<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaryPhaseWidget</class>
 <widget class="SGWidget" name="PrimaryPhaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true">QLabel
{
font-weight: bold;
font-size: 150%;
}</string>
     </property>
     <property name="text">
      <string>Primary Phase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sizeDistributionTab">
      <attribute name="title">
       <string>Size Distribution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="distHLayout">
         <item>
          <widget class="QComboBox" name="distributionTypeCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QwtPlot" name="m_SizeDistributionPlot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="widgetLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string extracomment="Mu: The average value of the lognormal grain size distribution. Note that this is the average value of log(ESD). This value is in the same arbitrary units of length as the plot, but is not the average Feature size"/>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Mu:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_Mu_SizeDistribution">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8" rowspan="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Bins to be Created:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="m_NumberBinsGenerated">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Bin Step Size:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QDoubleSpinBox" name="m_BinStepSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="maximum">
            <double>9999999.990000000223517</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="toolTip">
            <string extracomment="Sigma: The standard deviation of the lognormal grain size distribution. Note that this is not the standard deviation of the actual Feature size distribution, but of the log(ESD) values. This value controls the length of the “tails” of the distribution"/>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Sigma:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_Sigma_SizeDistribution">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>0.1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="m_MinSigmaCutOff">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_7">
           <property name="toolTip">
            <string>Allows the user to truncate the distribution to remove very large (or small) Features. These values show up as blue vertical lines on the plot and only the portion of the Feature size distribution between the two blue vertical lines can be sampled. The value entered by the user is the &quot;min&quot; or &quot;max&quot; variable in the equation shown.</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Min Sigma Cut Off Factor:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_4">
           <property name="toolTip">
            <string>Allows the user to truncate the distribution to remove very large (or small) Features. These values show up as blue vertical lines on the plot and only the portion of the Feature size distribution between the two blue vertical lines can be sampled. The value entered by the user is the &quot;min&quot; or &quot;max&quot; variable in the equation shown.</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Max Sigma Cut Off Factor:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="m_MaxSigmaCutOff">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; font-size:24pt; vertical-align:super;&quot;&gt;(μ + max * σ)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>1431324</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; font-size:24pt; vertical-align:super;&quot;&gt;(μ - min * σ)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_11">
           <property name="toolTip">
            <string>Entering the ESD and then pressing the &quot;Enter&quot; or &quot;Return&quot; key will automatically update the &quot;Mu&quot; value and the Size Distribution chart.</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
font-weight: bold;
font-size: 12px;
}</string>
           </property>
           <property name="text">
            <string>Feature ESD</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLineEdit" name="m_FeatureESD">
           <property name="text">
            <string>2.73</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Preset Statistic Models</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="microstructurePresetCombo">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_GenerateDefaultData">
           <property name="text">
            <string>Create Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="omega3DistributionTab">
      <attribute name="title">
       <string>Omega 3 Distributions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="StatsGenPlotWidget" name="m_Omega3Plot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shapeDistributionTab">
      <attribute name="title">
       <string>Shape Distributions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>779</width>
            <height>533</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="StatsGenPlotWidget" name="m_BOverAPlot" native="true"/>
           </item>
           <item>
            <widget class="StatsGenPlotWidget" name="m_COverAPlot" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="neighborDistributionTab">
      <attribute name="title">
       <string>Neighbor Distributions</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="StatsGenPlotWidget" name="m_NeighborPlot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="StatsGenODFWidget" name="m_ODFWidget">
      <attribute name="title">
       <string>ODF</string>
      </attribute>
     </widget>
     <widget class="SGAxisODFWidget" name="m_AxisODFWidget">
      <attribute name="title">
       <string>Axis ODF</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SGWidget</class>
   <extends>QWidget</extends>
   <header location="global">SGWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatsGenPlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenPlotWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatsGenODFWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenODFWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SGAxisODFWidget</class>
   <extends>QWidget</extends>
   <header location="global">SGAxisODFWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_Mu_SizeDistribution</tabstop>
  <tabstop>m_Sigma_SizeDistribution</tabstop>
  <tabstop>m_MinSigmaCutOff</tabstop>
  <tabstop>m_MaxSigmaCutOff</tabstop>
  <tabstop>m_BinStepSize</tabstop>
  <tabstop>microstructurePresetCombo</tabstop>
  <tabstop>m_GenerateDefaultData</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>distributionTypeCombo</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
